src/components/features/ctf-components/ctf-page-section/ctf-page-section.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { ComponentResolver } from '@src/components/shared/component-resolver';
+import { PageContainer } from '@src/components/templates/page-container';
+import { sectionTypes } from '@src/components/shared/global';
+
+interface PageSectionProps {
+  __typename: string;
+  sys: {
+    id: string;
+  }
+  componentsGrouped?: boolean;
+  componentsCollection?: any;
+  sectionType?: string;
+}
+
+
+export const CtfPageSection = (props: PageSectionProps) => {
+  const data = props.componentsCollection?.items as any[];
+
+  if (data && data.length > 0) {
+    console.log("data", data);
+    const testimonialCountObj = data.filter(item => item.__typename == sectionTypes.testimonialResourceType)
+    if (testimonialCountObj && testimonialCountObj.length > 0) {
+      data.map(testimonial => testimonial.maxItems = testimonialCountObj.length);
+    }
+
+    if (props.sectionType == sectionTypes.solutionSection) {
+      data.map(item => item.sectionType = props.sectionType);
+    }
+
+    if (props.sectionType == sectionTypes.hrSection) {
+      data.map(item => item.sectionType = props.sectionType);
+    }
+  }
+
+  return (
+    <PageContainer>
+      {props.componentsCollection && props.componentsCollection?.items.map(entry => (
+        <div key={entry!.sys.id}>
```

Comment:
Waarom de ! in deze variable gebruiken?

Reply:
Not sure why, it's copied from the template from CF. Its been removed now